Makefile.in

```diff
@@ -107,6 +107,8 @@ WXZLIB_OBJECTS =  \
 WXPNG_CFLAGS = -DNDEBUG $(__INC_ZLIB_p) $(____SHARED) $(CPPFLAGS) $(CFLAGS)
 WXPNG_OBJECTS =  \
 	wxpng_png.o \
+	wxpng_arm_init.o \
```

Comment:
I needed to add the files here so that other libraries could use the functions (wxx). The PNG library does some other stuff with all three files if present and is able to:  https://github.com/smanders/wxWidgets/blob/xp3.1.0/src/png/Makefile.am#L82
This is fixed in later versions of wxWidgets. 
Later versions of wxWidgets have all three of these files (which is where I got them from). They do not add the files here though because wxWidgets starts using cmake. 
Basically what I am saying is these changes are not necessary once (or if) we update wxWidgets.